.github/workflows/escalation-slack-notification.yml

```
@@ -0,0 +1,31 @@
+name: Escalation Slack Notification
+
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  on-escalation-label:
+    if: github.event.label.name == 'escalation'
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+      - name: On Escalation
```

It's useful to provide more context for people looking at the list of all actions and trying to figure out what happened.
```suggestion
      - name: Send escalated issue ${{ github.event.issue.number }} to Slack
```